src/queries/accessibilityOption.ts

```diff
@@ -0,0 +1 @@
+export type AccessibilityOption = { hidden?: boolean };
```

Comment:
1. How about putting this is `options.ts` file, and moving `TextMatchOptions` also into it. 
2. I would also suggest using `BaseOptions` name or something similar, as the idea here is that all queries accept that option.

Reply:
done